p/scripts/main.js

```diff
@@ -1280,14 +1279,20 @@ function init_crypto_form() {
 var $sidebar = null;
 var useJsScrollbar = true;
 try {
-	useJsScrollbar = !(parseInt(getComputedStyle(document.getElementById('sidebar'), '::-webkit-scrollbar').width) < 99);
```

Comment:
Ah, it is because my changes where manually overridden in this commit. Just a guess: what might have happened it that you pulled, but had a file opened in a local text editor, which did not get refreshed, so an older version was then saved.

Reply:
I am making a new detection to support Firefox 64.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2117/commits/68e30eb98bcaa4c167ae9f187bf6d1b829642128